andela/isengard-cfh
# Chore/148223873/grunt to gulp

#### What does this PR do?
This pull request is to convert the task runner from grunt to gulp.
#### Description of Task to be completed?
Grunt is an older task runner than Gulp; with Gulp, the syntax is much simpler to comprehend and it uses pipe streams which makes it inherently faster.
#### How should this be manually tested?
Run the following scripts to verify that the tasks handling them work
`npm install`: This runs the postinstall script, as the install script is not yet set.
`npm start`: This starts a series of tasks meant to get the application running. To verify that it works, go to port 3000 on your browser to run the app.
`npm test`: This runs the test suites. Some tests are currently failing, just as they were when run in the (former) grunt task runner.

#### Any background context you want to provide?
There are differences in the way tasks are created for grunt and for gulp. The gulp tasks might seem structurally dissimilar to the former (parallel) grunt tasks, but they basically do the same thing.
#### What are the relevant pivotal tracker stories?
Convert from Grunt to Gulp taskrunner
#### Screenshots (if appropriate)
None
#### Questions:
None
